kustomize/overlays/minikube/manager/kustomization.yaml

```diff
@@ -7,6 +7,13 @@ secretGenerator:
   - EVENT_BRIDGE_KAFKA_BOOTSTRAP_SERVERS=my-cluster-kafka-bootstrap.kafka:9092
   - EVENT_BRIDGE_KAFKA_CLIENT_ID=client-id
   - EVENT_BRIDGE_KAFKA_CLIENT_SECRET=secret
+  - EVENT_BRIDGE_KAFKA_SECURITY_PROTOCOL=SASL_SSL
+  - EVENT_BRIDGE_RHOAS_INSTANCE_API_HOST=instance-api-host
+  - EVENT_BRIDGE_RHOAS_SSO_MAS_AUTH_SERVER_URL=https://identity.api.openshift.com/auth/realms/rhoas
+  - EVENT_BRIDGE_RHOAS_SSO_MAS_CLIENT_ID=client-id
+  - EVENT_BRIDGE_RHOAS_SSO_MAS_CLIENT_SECRET=secret
+  - RHOAS_OPS_ACCOUNT_CLIENT_ID=client-id
+  - RHOAS_TOPIC_PREFIX=ob-
```

Comment:
Should this be aligned with the other Kafka properties e.g `EVENT_BRIDGE_KAFKA_TOPIC_PREFIX` ?

Reply:
What emerged by itself from previous PRs is that properties starting with `EVENT_BRIDGE_RHOAS*` belongs to `rhoas-client` while those starting with `RHOAS_*` are specific to the `manager`.
I think we should keep this pattern and fix it later in a dedicated task if needed, unless there's some kind of guideline for property naming that I'm not aware of.
@robpblake wdyt?

Reply:
Refactored as you suggested